appinventor/components/src/com/google/appinventor/components/runtime/Image.java

```diff
@@ -107,10 +107,12 @@ public void Picture(String path) {
 
   @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_BOOLEAN,
       defaultValue = "False")
-  @SimpleProperty
+  @SimpleProperty(description = "Specifies whether the image should be resized to match the size                                 of the ImageView.")
```

Comment:
is there an issue with the indentation here or is it github showing things *funnily*?